plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Repositories.js

```diff
@@ -371,6 +375,45 @@ Ext.define('NX.coreui.controller.Repositories', {
           scope: button
         }
     );
+  },
+
+  /**
+   * @override
+   * Rebuild Repository index.
+   */
+  rebuildIndex: function(model) {
+    var me = this;
+
+    NX.direct.coreui_Repository.rebuildIndex(model.getId(), function(response) {
+      me.getStore('Repository').load();
```

Comment:
why do we need to reload? None of the repository data is mutated, although if/when we provide info on re-indexing progress that might change. In any case it likely should be gated by whether or not we're successful.